libs/core/src/main/java/org/opensearch/core/common/Strings.java

```diff
@@ -35,18 +36,6 @@ public class Strings {
     public static final String UNKNOWN_UUID_VALUE = "_na_";
     public static final String[] EMPTY_ARRAY = new String[0];
 
-    /**
-     * Split the specified string by commas to an array.
-     *
-     * @param s the string to split
-     * @return the array of split values
-     * @see String#split(String)
-     */
-    public static String[] splitStringByCommaToArray(final String s) {
```

Comment:
Did the `splitStringByCommaToArray` and `delimitedListToStringArray` methods just move to a different location within the same class file?

Reply:
Yes, that's correct